### PR TITLE
#434 updated references, added missing iSAQB glossary etc

### PR DESCRIPTION
**Pull request by gernotstarke**
if one of you could please take a brief look... trivial change.

### CODE REVIEW

**Review by CodingJenka — APPROVED** [review]

**Review by mahboubagharbi — DISMISSED** [review]

File: docs/09-references/00-references.adoc [hunk 1]
```diff
@@ -49,6 +49,8 @@
 - [[[hombergs,Hombergs 2024]]] Hombergs, Tom: Get Your Hands Dirty on Clean Architecture, Packt, 2nd edition 2024.
 
 // I
+- [[[isaqbdownloads,iSAQB Downloads]]] iSAQB public download site.  https://public.isaqb.org. Contains curriculae and mock-examination.
+- [[[isaqbglossary,iSAQB Glossary]]] Gernot Starke et. al. iSAQB Glossary of Software Architecture Terminology. Freely available https://leanpub.com/isaqbglossary.
```
Comment (alxlo) [inline docs/09-references/00-references.adoc:53]:
```suggestion
- [[[isaqbglossary,iSAQB Glossary]]] Gernot Starke et. al. iSAQB Glossary of Software Architecture Terminology. Freely available from https://leanpub.com/isaqbglossary or its source repository https://github.com/isaqb-org/glossary/releases
```

**Review by alxlo — COMMENTED** [review]
I'd rather point to Leanpub AND the Github repo for the glossary, see suggestion.

File: docs/09-references/00-references.adoc [hunk 2]
```diff
@@ -49,6 +49,8 @@
 - [[[hombergs,Hombergs 2024]]] Hombergs, Tom: Get Your Hands Dirty on Clean Architecture, Packt, 2nd edition 2024.
 
 // I
+- [[[isaqbdownloads,iSAQB Downloads]]] iSAQB public download site.  https://public.isaqb.org. Contains curriculae and mock-examination.
```
Comment (programming-wolf) [inline docs/09-references/00-references.adoc:52]: *curricula (curriculae is wrong, as curriculum is the singular form 🙂 )

**Review by programming-wolf — CHANGES_REQUESTED** [review]
Please fix that one typo, then we can merge this PR 🙂

**Comment by programming-wolf** [comment]
Now there are merge conflcits..

**Pull request merged**